RocketChat/Rocket.Chat
# fix: New messages export overwrites previous one from the same day when using Amazon S3

## Proposed changes (including videos or screenshots)

 - Include unique id in message export zip files' names to avoid replacement instead of insertion on external storage services (due to name collision).

## Issue(s)


## Steps to test or reproduce

Set Amazon S3 as the Storage Type and sequentially export multiple rooms as files.

**Current behavior:** by accessing the oldest export sent by email and clicking the link, the downloaded file is actually related to the latest export.

 **Expected behavior:** each downloaded file should contain its corresponding export, no overwrites should be made.

## Further comments

[SUP-522](https://rocketchat.atlassian.net/browse/SUP-522)

[SUP-522]: https://rocketchat.atlassian.net/browse/SUP-522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
